.github/workflows/jest-testing.yml

```
@@ -27,14 +27,19 @@ jobs:
         with:
           version: nightly
 
+      - name: Install Bun
+        run: |
+          curl -fsSL https://bun.sh/install | bash
+          echo "$HOME/.bun/bin" >> $GITHUB_PATH
+
       - name: Install dependencies
-        run: yarn install
+        run: bun install
 
       - name: Build
-        run: yarn build
+        run: bun run build
 
       - name: Start Anvil
-        run: yarn test:anvil &
+        run: bun test:anvil &
```

```suggestion
        run: bun run test:anvil &
```

Always prefer to add `run` to avoid using `bun` built-ins by accident.